Launcher/Classes/LocalizationHandler.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+
+class LocalizationHandler {
+    
+    let applicationStateHandler = ApplicationStateHandler()
+    let fileManager = FileManager.default
+    var jsonFilePaths: [String] = []
+    var localizedStringsFilePaths: [String] = []
+    
+    init() {
+        jsonFilePaths = allFiles(withFileExtension: "json")
+        localizedStringsFilePaths = allFiles(withFileExtension: "strings")
+    }
+    
+    func keys(for string: String) -> [String] {
```

Comment:
This might be easier to understand when called `localizationKeys(for value: String)`

Reply:
Also the header docs could show the priority of `keysForLocalizedStrings` over `keysForJson `.